display_list/display_list_image_filter.h

```diff
@@ -137,6 +152,10 @@ class DlImageFilter
                                            const SkMatrix& ctm,
                                            SkIRect& input_bounds) const = 0;
 
+  virtual MatrixCapability get_matrix_capability() const {
+    return MatrixCapability::kScaleTranslate;
```

Comment:
Where did you get information about the compatibility here?

Reply:
In Skia, the ImageFilter has a method `onGetCTMCapability` that will return the `MatrixCapability`.